filters/include/pcl/filters/impl/normal_space.hpp

```diff
@@ -243,7 +243,7 @@ pcl::NormalSpaceSampling<PointT, NormalT>::applyFilter (std::vector<int> &indice
       // Picking up a sample at random from jth bin
       do
       {
-        random_index = static_cast<unsigned int> ((*rng_uniform_distribution_) () % M);
+        random_index = static_cast<unsigned int> (uniform_distrib (rng) % M);
```

Comment:
```suggestion
        random_index = uniform_distrib (rng) % M;
```

Or is the cast required because of the `%` operator?

Reply:
Shouldn't because `uniform_distrib` returns an `unsigned int` and `M`is an `unsigned int` too.